bundles/psp-v2022/psp-flexvolume-drivers.yaml

```diff
@@ -39,7 +39,9 @@ spec:
           - ""
         kinds:
           - Pod
-  parameters:
-    allowedFlexVolumes: #[]
-      - driver: example/lvm
-      - driver: example/cifs
+  # K8sPSPFlexVolumes documentation and parameters:
+  # https://cloud.google.com/anthos-config-management/docs/latest/reference/constraint-template-library#k8spspflexvolumes
+  parameters: {}
+  #     allowedFlexVolumes: #[]
```

Comment:
Comment blocks? Are they needed?

Reply:
Yeah - They are to help with the Migration workflow: https://cloud.google.com/blog/products/containers-kubernetes/migrate-psp-policies-policy-bundle